📝 Add Console steps for source integrations (#2689)
platformsh/platformsh-docs: docs/src/integrations/source/gitlab.md

{{< /note >}}

## 2. Enable the integration

To enable the integration, use either the [CLI](../../administration/cli/_index.md)
or the [Console](../../administration/web/_index.md).

{{< codetabs >}}
---

> title:Using the CLI

Reviewer: 🛠️ 

```suggestion
title=Using the CLI
```

Note that the `--prune-branches` option depends on `--fetch-branches` being enabled.
If `--fetch-branches` is disabled, `--prune-branches` is automatically be set to false, even if specifically set to true.

<--->
---

> title:In the Console

Reviewer: 🛠️ 

```suggestion
title=In the Console
```

file=none
highlight=false
---

1. Select the project where you want to enable the integration.
2. Click {{< icon settings >}} **Settings**.
3. Under **Project settings**, click **Integrations**.
4. Click **+ Add integration**.
5. On the GitLab integration, click **+ Add**.
6. Add the token you generated.
7. Optional: If your GitLab instance is not hosted at `https://gitlab.com`, enter its base URL.
8. Choose the repository to use for the project.
9. Check that the other options match what you want.
10. Click **Add integration**.

{{< /codetabs >}}

## 3. Validate the integration

Verify that your integration is functioning properly [using the CLI](../overview.md#validating-integrations):

```bash
platform integration:validate
```

### Add the webhook

If the integration was added with correct permissions, the necessary webhook is added automatically.
If you see the message `Failed to read or write webhooks`, you need to add a webhook manually:

1. Get the webhook URL by running `platform integration:get`
2. Copy the `hook_url`.
3. Go to your GitLab repository and click Settings > Webhooks.
4. Paste the hook URL.

>    In the Triggers section choose Push events, Tag push events and Merge Request events.

Author: ```suggestion
   In the **Trigger** section select **Push events**, **Tag push events**, and **Merge request events**.
```
